.github/header-checker-lint.yml

```diff
@@ -0,0 +1,5 @@
+allowedLicenses:
+  - Apache-2.0
+sourceFileExtensions:
+  - java
+  - sh
```

Comment:
@jsuereth do we need copyright headers on `.gradle` and `.sh`? Guidelines just say "all source files", what do you think?

Reply:
I think it's a good idea to include them.